diff --git a/node_modules/@types/mssql/index.d.ts b/node_modules/@types/mssql/index.d.ts
index 6be140a..27afed0 100755
--- a/node_modules/@types/mssql/index.d.ts
+++ b/node_modules/@types/mssql/index.d.ts
@@ -138,6 +138,9 @@ export interface IResult<T> {
     recordset: IRecordSet<T>;
     rowsAffected: number[],
     output: { [key: string]: any };
+
+    // patch
+    columns: IColumnMetadata[];
 }
 
 export interface IBulkResult {
@@ -342,6 +345,9 @@ export declare class Request extends events.EventEmitter {
     public cancel(): void;
     public pause(): boolean;
     public resume(): boolean;
+
+    // patch
+    public arrayRowMode?: boolean | undefined;    
 }
 
 export declare class RequestError extends MSSQLError {
